object_tracker/object_tracker.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from pytracking.evaluation import Tracker
+from object_tracker.match import Matcher
+
+logging.getLogger().setLevel(logging.INFO)
+
+
+class ObjectTracker:
+
+    MAX_COUNTER = 10
+
+    def __init__(self, run_optical_flow=False, use_orb=False, tracker_run_iter=3):
+        self._run_optical_flow = run_optical_flow
+        self._tracker_run_iter = max(tracker_run_iter, 1)
+        self._tracker_counter = 0
+        self._tracker = Tracker("dimp", "dimp18")
+        self._match = Matcher(use_orb)
+        self.init = False
+
+    def run_frame(self, img):
+        if not self.init:
+            return
+        if self._tracker_counter > ObjectTracker.MAX_COUNTER:
+            self._tracker_counter = 0
+        self._tracker_counter += 1
+        orig = img.copy()
+        optical_flow_output = None
+        if self._run_optical_flow:
+            optical_flow_output = self._match(orig)
```

Comment:
The matcher is CPU intense, I don't think we should calculate it for free when it is dimp's turn

Reply:
If we'll implement OF as a backup for hidden target cases we may want to change the logic, but for now I don't think we need it.